web/html/src/manager/content-management/list-filters/list-filters.tsx

```diff
@@ -62,6 +113,22 @@ const ListFilters = (props: Props) => {
     </div>
   );
 
+  const unusedFilter = (
+    <button className="btn-link" onClick={onSelectUnused}>
+      {t("Select unused")}
+    </button>
+  );
+
+  const deleteSelected = (
+    <button
+      className={`btn ${selectedIdentifiers.length ? "btn-danger" : "btn-disabled"}`}
+      disabled={!selectedIdentifiers.length}
+      onClick={deleteSelectedRows}
+    >
+      {t("Delete selected")}
+    </button>
```

Comment:
Couldn't you use our `Button` component here?

Reply:
Sure could, somehow didn't cross my mind. Thanks! 👍